jsk_robot_common/jsk_robot_startup/README.md

```diff
@@ -187,6 +187,15 @@ This node shuts down or reboots the robot itself according to the rostopic. Note
 
   Input topic that trigger reboot
 
+* `shutdown_unchecked` (`std_msgs/Empty`)
```

Comment:
Does this node shoutdown computer if we set `~input_condition` to `None`? if so , I think  we do not need shutdown_unchecked. Other part is very nice.

Reply:
We may want to run shutdown when the battery is charged. For example, blackout.
So shutdown_unchecked (shutdown with check) and shutdown (force shutdown) are required.

Reply:
1) in that case, we can ssh and run `shutdown`. Of course if robot can understand it is black out or earthquack, fires, then it is different story
2) Even if you wan to force shoutdown, set `~input_condition` to `None` and send `shutdown`

Reply:
I see. I understand it.
I modified the code to disable `shutdown_unchecked`.
https://github.com/jsk-ros-pkg/jsk_robot/pull/1571/commits/9d359f0a80e1fc5cbcb0b0c0b5fe8cbcdb9f6d48